qml/Ping1DVisualizer.qml

```diff
@@ -52,7 +53,7 @@ Item {
                 transform: Rotation { origin.x: height/2; angle: 90}
                 gradient: Gradient {
                     GradientStop { position: 0.3; color: "transparent" }
-                    GradientStop { position: 0.5; color: Style.primaryColor } // Not working with material
+                    GradientStop { position: 0.5; color: StyleManager.secondaryColor } // Not working with material
```

Comment:
is the comment still relevant?

Reply:
Yep.

Reply:
Because they are not working with Material. As the comment says.
Probably a bug in the signal propagation of Material, I'm not sure.